README.md

```diff
@@ -1,75 +1,88 @@
+Allows add-on developers to build [Shield Study](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies) ([Normandy](https://wiki.mozilla.org/Firefox/Shield#Normandy_-_User_Profile_Matching_and_Recipe_Deployment)) compatible add-ons without having to think very much.
 
+## What You are Building
 
+* You are building a [LEGACY ADD-ON](https://developer.mozilla.org/Add-ons/Legacy_add_ons). To deploy these after 57, you will need the magic special signing.
+* Web Extensions are not strong enough [yet](https://github.com/mozilla/shield-studies-addon-utils/issues/45).
+* Jetpack / addon-sdk is NOT AT ALL SUPPORTED since v4 of this utils library.
 
-## install
+## Get started
+
+Check out [mozilla/shield-studies-addon-template/](https://github.com/mozilla/shield-studies-addon-template/) to get started with an example study where shield-studies-addon-utils is already installed and configured.
+
+## Installing the `StudyUtils.jsm` in your add-on
 
 ```
 npm install --save-dev shield-studies-addon-utils
 ```
 
-Copy the file to somewhere useful in your addon.
-
-
-## Tutorial and Full Usage
-
-[fully worked tutorial - How To Shield Study](./howToShieldStudy.md)
-
-## Example embedded web extension study
-
-See [examples](https://github.com/mozilla/shield-studies-addon-utils/tree/master/examples) directory.
+Copy `dist/StudyUtils.jsm` to your `addon` source directory, where it will be zipped up.
 
 ## Summary
 
 ### Design Case
 
 Your Study is:
 
-- side-by-side variations (1 or more)
+* side-by-side variations (1 or more)
 
 ### Benefits
 
 Using this, you get this analysis FOR FREE (and it's fast!)
 
-- Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
-
-
-## Development
+* Branch x channel x VARIATION x experiment-id x PHASE (install, reject, alive etc) using UNIFIED TELEMETRY
 
-- open an issue
-- hack and file a PR
+## Engineering and Process
 
+* [Shield article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield)
+* [Shield Studies article on Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies)
+* [mozilla/shield-studies-addon-template/](https://github.com/mozilla/shield-studies-addon-template/)
+* [Current work-in-progress docs and launch process](https://github.com/mozilla/shield-studies-addon-utils/issues/93)
+* [Long, rambling engineering docs](./docs/engineering.md)
+* Come to slack: #shield
 
 ## Gotchas, Opinions, Side Effects, and Misfeatures
 
-1.  No handling of 'timers'.  No saved state at all (including the variation name), unless you handle it yourself.
+1.  No handling of 'timers'. No saved state at all (including the variation name), unless you handle it yourself.
 
 2.  No 'running' pings in v4 (yet).
 
-3.  User disable also uninstalls (and cleans up)
+3.  User disable also uninstalls (and cleans up).
+
+## Development on the Utils
+
+* open an issue
+* hack and file a PR
+
+## History of major versions
```

Comment:
This is great! I think the next version should take advantage of the "[Releases](https://help.github.com/articles/creating-releases/)" tab in the repo.